src/index.html

```diff
@@ -11,7 +11,186 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-field">
+        <legend>
+          Personal information
+        </legend>
+        <label class="input">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML. Check it everywhere
```suggestion
        </legend>
        
        <label class="input">
```